app/src/main/java/com/example/ithaca_transit_android_v2/NetworkingAdapters.kt

```diff
@@ -1,22 +1,28 @@
 package com.example.ithaca_transit_android_v2
 
+import android.util.Log
+import androidx.annotation.Nullable
 import com.example.ithaca_transit_android_v2.models.*
-import com.squareup.moshi.JsonClass
-import com.squareup.moshi.FromJson
-import com.squareup.moshi.ToJson
-import com.squareup.moshi.Json
-import com.squareup.moshi.JsonWriter
+import com.squareup.moshi.*
 import java.text.SimpleDateFormat
-import java.util.Date
-import java.util.Calendar
-import java.util.Locale
-import java.util.concurrent.TimeUnit
+import java.util.*
+import kotlin.collections.ArrayList
+import kotlin.collections.List
+import kotlin.collections.map
 
 @JsonClass(generateAdapter = true)
 class LocationAdapter {
+
     class DataLocation(
-        val data: List<JsonLocation>
+        val success: Boolean,
```

Comment:
lowkey forgot how moshi works. is the `success` param necessary?

Reply:
yea looks like its not needed